javascript-timers/interval.js

```diff
@@ -0,0 +1,14 @@
+var $h1 = document.querySelector('h1');
+setInterval(countdown, 1000);
+
+function countdown(second) {
+  if ($h1.textContent === '4') {
+    $h1.textContent = '3';
+  } else if ($h1.textContent === '3') {
+    $h1.textContent = '2';
+  } else if ($h1.textContent === '2') {
+    $h1.textContent = '1';
+  } else if ($h1.textContent === '1') {
```

Comment:
If you create a variable to decrement over time, I think you could do this without individual code blocks for each number

Reply:
Also, you didn't clear your interval, so the code is still running over and over again 👀